lib/bcoin.js

```diff
@@ -72,16 +73,28 @@ class LedgerBcoin {
     return this;
   }
 
+  /**
+   * Get firmware version.
+   * @returns {Object}
+   */
+
+  async getFirmwareVersion() {
+    return this.ledger.getFirmwareVersion();
+  }
+
   /**
    * Get public key.
    * @async
    * @param {(Number[]|String)} - Full derivation path
+   * @param {Boolean} [parentFingerPrint = false] - Note:
+   *  This will make it run slower.
```

Comment:
Is there really that big of a difference in performance such that we shouldn't just do it every time?

Reply:
Twice as slow, so it depends. If you don't need to get fingerprint, then dont :) (e.g. get 1000 pubkeys)

Reply:
I guess it depends on the usecase, if you strictly need pubkeys/addresses vs xpubs, but it makes sense to allow it to be off for that usecase